tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -473,15 +543,17 @@ Status FilterPipeline::run_forward(
   return Status::Ok();
 }
 
+template<bool HasBitSortFilter>
 Status FilterPipeline::run_reverse_chunk_range(
-    stats::Stats* const reader_stats,
-    Tile* const tile,
-    const ChunkData& chunk_data,
-    const uint64_t min_chunk_index,
-    const uint64_t max_chunk_index,
-    uint64_t concurrency_level,
-    const Config& config) const {
-  // Run each chunk through the entire pipeline.
+      stats::Stats* const reader_stats,
+      Tile* const tile,
+      const ChunkData& chunk_data,
+      const uint64_t min_chunk_index,
+      const uint64_t max_chunk_index,
+      uint64_t concurrency_level,
+      const Config& config,
+      OptionalRef<BitSortFilterMetadataType> bitsort_metadata) const {
```

Comment:
Let's get rid of this OptionalRef and add a templated argument to the API (possibly support_data)... This should also get rid of template<bool HasBitSortFilter>. Ultimately, I would like the filter pipeline to look like a "passthrough" for all this support data coming down from the reader.